src/Elsa.Dashboard/package.json

```diff
@@ -29,7 +29,6 @@
     "unit-test": "jest"
   },
   "dependencies": {
-    "@elsa-workflows/elsa-workflows-studio": "^2.8.2",
```

Comment:
is this intended?

Reply:
actually not being used because we are not importing anything from elsa directly